ansible/roles/ocp4-workload-integreatly/defaults/main.yml

```diff
@@ -21,7 +21,17 @@ silent: False
 # (if defined).
 #
 # The logic in the workload should only use the dictionary "<role name>".
```

Comment:
Based on this comment, should these variables all be under the `ocp4_workload_integreatly_defaults` dict?

Reply:
I didn't think we'd want those as override params :thinking:

Reply:
Yep, ignore